xcode/Subconscious/Shared/Models/MemoEditorDetailResponse.swift

```diff
@@ -11,7 +11,6 @@ import Foundation
 struct MemoEditorDetailResponse: Hashable {
     var saveState: SaveState
     var entry: MemoEntry
-    var backlinks: [EntryStub] = []
```

Comment:
Since this struct only carries an entry now, should we just respond with a MemoEntry?

Reply:
I think we still want to attach `saveState` here, so the struct makes sense?

Reply:
Oops, missed that. You're right.